README.md

```diff
@@ -1,23 +1,22 @@
 # [One Dark Pro](https://marketplace.visualstudio.com/items?itemName=zhuangtongfa.Material-theme)
 
-Atom's iconic One Dark theme, and one of the most downloaded [themes](https://marketplace.visualstudio.com/search?target=VSCode&category=Themes&sortBy=Downloads) for VS Code!
+Atom's iconic One Dark theme, and one of the most downloaded [themes](https://marketplace.visualstudio.com/search?target=VSCode&category=Themes&sortBy=Installs) for VS Code!
```

Comment:
The description clearly specifies downloaded themes but this changes the number to the number of installs.

Reply:
good point! Thanks

Reply:
link:
https://marketplace.visualstudio.com/search?target=VSCode&category=Themes&sortBy=Downloads
is redirected to: 
https://marketplace.visualstudio.com/search?target=VSCode&category=Themes&sortBy=Relevance

which is not relevant enough. I'm changing the description